tests/bud.bats

```diff
@@ -6946,3 +6946,23 @@ _EOF
   run_buildah run testctr -- sh -c 'cd podman-tag && git ls-remote --tags origin v5.0.0^{} | cut -f1'
   assert "$output" = "$local_head_hash"
 }
+
+@test "bud with ADD with file with IMA xattr" {
+  _prefetch alpine
+
+  local contextdir=${TEST_SCRATCH_DIR}/add-ima
+  mkdir -p $contextdir
+  cat > $contextdir/Dockerfile <<EOF
+FROM alpine
+ADD /usr/libexec/catatonit/catatonit /bin/catatonit
+EOF
+
+  # Verify that /usr/libexec/catatonit/catatonit both exists and has an IMA xattr
+  if getfattr -d -m 'security.ima' /usr/libexec/catatonit/catatonit | grep -q ima; then
+    skip "catatonit does not exist or does not have an appropriate xattr"
```

Comment:
I'm not a big fan of ambiguous `skips` or test failures. If I were looking at this skip, I'd want to know more.